website/docs/docs/tutorials/primary-api.md

```diff
@@ -0,0 +1,751 @@
+provide the server rendered state to our store via the `initialState`
+configuration option.
+
+```javascript
+const store = createStore(model, {
+  initialState: serverRenderedState,
+});
+```
+
+### Some fun facts about the store
+
+#### 1. It's a Redux store
+
+The store instance that is created is in fact just a Redux store (with a few
+enhancements added). Therefore you could use it with anything that expects a
+Redux store.
+
+For example, the `react-redux` `Provider`:
+
+```javascript
+import { Provider } from 'react-redux';
+import store from './my-easy-peasy-store';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root'),
+);
+```
+
+#### 2. It's a Redux store
+
+It's worth stressing this point, as you can use all the
+[APIs](https://redux.js.org/api/store) of a standard
+[Redux store](https://redux.js.org/api/store).
+
+```javascript
+store.subscribe(() => {
+  console.log('A state changed occurred');
+});
+```
+
+#### 3. It's not just a Redux store
+
+😅
+
+Ok, so we have made a [few enhancements to the API](/docs/api/store.html),
+extending the standard [Redux store API](https://redux.js.org/api/store) with
+some Easy Peasy specific APIs. An example of one below.
+
+```javascript
+import store from './my-easy-peasy-store';
+
+store.getActions().addTodo('Learn Easy Peasy');
+```
+
+In the above we are using one of the [extended APIs](/docs/api/store.html) to
+get the actions defined in our model. We are then dispatching the `addTodo`
+action, providing it a payload of `"Learn Easy Peasy"`.
+
+You can read more about the extended API [here](/docs/api/store.html).
+
+## Connecting the Store
+
+To utilize the store within your React application you need to wrap your
+application with the [StoreProvider](/docs/api/store-provider.html) component,
+providing the store as prop.
+
+```javascript
+import { StoreProvider } from 'easy-peasy';
+import store from './my-easy-peasy-store';
+
+ReactDOM.render(
+  <StoreProvider store={store}>
+    <App />
+  </StoreProvider>,
+  document.getElementById('root'),
+);
+```
+
+This is a very similar experience when compared to using Redux.
+
+## Using the Store
+
+Easy Peasy ships with a variety of hooks allowing for convenient interaction
+with the store from your components.
+
+### The `useStoreState` hook
+
+To consume state within your components utilize the
+[useStoreState](/docs/api/use-store-state.md) hook.
+
+```javascript
+import { useStoreState } from 'easy-peasy';
+
+function Todos() {
+  const todos = useStoreState((state) => state.todos);
+  return <TodoList todos={todos} />;
+}
+```
+
+The `useStoreState` hook accepts a selector function which should resolve the
+state that your component needs.
+
+It's completely reasonable to use the `useStoreState` multiple times within a
+component to resolve all the various pieces of state you might require.
+
+```javascript
```

Comment:
is this the good code example ? It's the same as previous one and there is only one useStoreState..